src/main/CommandHandler.java

```diff
@@ -0,0 +1,16 @@
+package main;
+
+public class CommandHandler {
+
+	
```

Comment:
you want one method to accept all commands. Plus you will accept a String, not an array
`public void processCommand(final String command)`

Reply:
need to add actual implementation as well and "connect" it to actual do work